src/main/scala/org/clulab/habitus/actions/HabitusActions.scala

```diff
@@ -152,4 +163,29 @@ class HabitusActions extends Actions {
     }
     toReturn
   }
+
+  def FertilizerEventToRelation(mentions: Seq[Mention]): Seq[Mention] = {
+    // creates a binary fertilizer event from a rule where the trigger was a token that should also serve as a variable in an assignment event
+    val toReturn = new ArrayBuffer[Mention]()
+    for (m <- mentions) {
```

Comment:
This looks like a straight 1-to-1 conversion, so  a map:
```scala
    mentions.map { m =>
      ....
      new RelationMention(
        ...
      )
    }
```

Reply:
that does look much better!